bundles/org.openhab.binding.knx/src/test/java/org/openhab/binding/knx/internal/itests/Back2BackTest.java

```diff
@@ -0,0 +1,557 @@
+        final int dimensionlessCounter = 0b10111010;
+        helper("229.001", new byte[] { 0, 0, 0, 0, (byte) dimensionlessCounter, 0 }, new DecimalType(0));
+    }
+
+    @Test
+    void testColorDpts() {
+        // HSB
+        helper("232.600", new byte[] { 123, 45, 67 }, ColorUtil.rgbToHsb(new int[] { 123, 45, 67 }));
+        // RGB, MDT specific
+        helper("232.60000", new byte[] { 123, 45, 67 }, new HSBType("173.6, 17.6, 26.3"));
+
+        // xyY
+        int x = (int) (14.65 * 65535.0 / 100.0);
+        int y = (int) (11.56 * 65535.0 / 100.0);
+        // encoding is always xy and brightness (C+B, 0x03), do not test other combinations
+        helper("242.600", new byte[] { (byte) ((x >> 8) & 0xff), (byte) (x & 0xff), (byte) ((y >> 8) & 0xff),
+                (byte) (y & 0xff), (byte) 0x28, 0x3 }, new HSBType("220,90,50"), new byte[] { 0, 8, 0, 8, 0, 0 },
+                new byte[0]);
+        // TODO check brightness
+
+        // RGBW, only RGB part
+        helper("251.600", new byte[] { 0x26, 0x2b, 0x31, 0x00, 0x00, 0x0e }, new HSBType("207, 23, 19"),
+                new byte[] { 1, 1, 1, 0, 0, 0 }, new byte[0]);
+        // RGBW, only RGB part
+        helper("251.600", new byte[] { (byte) 0xff, (byte) 0xff, (byte) 0xff, 0x00, 0x00, 0x0e },
+                new HSBType("0, 0, 100"), new byte[] { 1, 1, 1, 0, 0, 0 }, new byte[0]);
+    }
+
+    @Test
+    void testColorTransitionDpts() {
+        // DPT 243.600 DPT_Colour_Transition_xyY
+        // time(2) y(2) x(2), %brightness(1), flags(1)
+        helper("243.600", new byte[] { 0, 5, 0x7F, 0, (byte) 0xfe, 0, 42, 3 },
+                new StringType("(0.9922, 0.4961) 16.5 % 0.5 s"));
+        // DPT 249.600 DPT_Brightness_Colour_Temperature_Transition
+        // time(2) colortemp(2), brightness(1), flags(1)
+        helper("249.600", new byte[] { 0, 5, 0, 40, 127, 7 }, new StringType("49.8 % 40 K 0.5 s"));
+        // DPT 250.600 DPT_Brightness_Colour_Temperature_Control
+        // cct(1) cb(1) flags(1)
+        helper("250.600", new byte[] { 0x0f, 0x0e, 3 }, new StringType("CT increase 7 steps BRT increase 6 steps"));
+        // DPT 252.600 DPT_Relative_Control_RGBW
+        // r(1) g(1) b(1) w(1) flags(1)
+        helper("252.600", new byte[] { 0x0f, 0x0e, 0x0d, 0x0c, 0x0f },
+                new StringType("R increase 7 steps G increase 6 steps B increase 5 steps W increase 4 steps"));
+        // DPT 253.600 DPT_Relative_Control_xyY
+        // cs(1) ct(1) cb(1) flags(1)
+        helper("253.600", new byte[] { 0x0f, 0x0e, 0x0d, 0x7 },
+                new StringType("x increase 7 steps y increase 6 steps Y increase 5 steps"));
+        // DPT 254.600 DPT_Relative_Control_RGB
+        // cr(1) cg(1) cb(1)
+        helper("254.600", new byte[] { 0x0f, 0x0e, 0x0d },
+                new StringType("R increase 7 steps G increase 6 steps B increase 5 steps"));
+    }
+
+    @Test
+    @AfterAll
```

Comment:
I don't think `@AfterAll` is the correct annotation here. What do you wanzt to achieve with that?

Reply:
If I got this correctly, 
> @ AfterAll is used to signal that the annotated method should be executed after all tests in the current test class. 

This method should run after all other tests, as each test adds all tested DPTs to ```dptTested```.